tests/dom-shim-tests/dom-shim-test.js

```diff
@@ -1,3 +1,7 @@
+global.navigator = {
+  userAgent: ''
```

Comment:
Hmm.. I would have thought it would be Chrome since PhantomJS is running these files. Sounds good though!